app/layout.tsx

```diff
@@ -1,26 +1,38 @@
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import type {Metadata} from "next";
+import {Inter} from "next/font/google";
```

Comment:
I need to update my IDE config to match the formatting ya'll are using. I'll do that and update this PR before merging.